Database-basic/ER-diagram.drawio.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mxfile host="app.diagrams.net" agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36" version="26.0.3"><diagram name="Page-1" id="JJF7eU91X1c3ni8JyWAB">7Ztdb+I4FIZ/DdLuRVGcxPm4LLSd3dkZbdVqNbN7UxnigqcmZh1TYH792IlDEidAoGTKSFRIxSc+juPz+smxE3rOcLb6wNF8+plFmPZsK1r1nJuebdsW8OU/ZVlnFuhYmWHCSZSZQGF4JN+xNubVFiTCSaWiYIwKMq8axyyO8VhUbIhztqxWe2a0etY5muCa4XGMaN36hURimlkDaBX2PzCZTPMzA0sfmaG8sjYkUxSxZcnk3PacIWdMZN9mqyGmavDyccn87rYc3XSM41i0cRiJUXL/lV3z21vqf/l2dzd5+X7lZK28IrrQF/xPgnmieyzW+TAkSzKjKJalwTOLxaM+YsnyeEpo9Amt2UJ1IxFo/JKXBlPGyXdZH1F5CEiDPMyFjrLtqdYIpUNGGZeGmKUnKJweVWP6NBwn0u0+v1xgmD6jVaXiJ5SIvIOMUjRPyCjtsnKcIT4h8YAJwWa60nJKBH6co7Gqs5RaVh0Rs7zf9cHW4/+KucCrkkkP/gfMZljwtayymQuZh54IIcyKy0JVINBVpmVF5YJCWsmTTctFsOUXHe8DYu/WYr+QsX+SPrb12/1fv/dsj6ogRuRVfp2or2mFGM1wfkyeuHS4wYMzinWTd0WTe93GHCOBoycktrkYApVBEKm+OHvBhqAaNIYomcSySPGzclNRJHLSX2uzYCr+iZQDiSef0jo3bmF50OFRJiZ9n2k6sackinCcXrRAAo02U2TOSCzS8MGB/MiADq0+7EHZ8aEsg6IsP6o6F0MWy2tBJJUblmJeYiXow3W6c9rvF69Wq5yrrdTqux2J1a+J9W8eXUjVDamCKqo2CNrLKtAVq4Ja+BugwZQi6vzaC5sS9kqMGnHTZW9DaQdUsiPZ1dpJMCU6lcEkW2FXcxox9pIIxluw9QLKN4DSbz13DgRl0BUo8ylYmioDKZYLJ0/PST80Urq2mHS6wiSw23BSwWMfJgURVDEMzdS4xaNknoHILJtto4WQ0T4i41tjxJ/mixElyRRHrd3mnIwPyEbx5gxP6oQXZnbAzA2BTg5NYHVGzfpCeJDfYuuqv6D01Ch1vSpKNyno+y2PQX193CIR2wrUEnTvGtfWRUsHLbAbTLIVqYydDVyAd0rg/XrLaQBr4r5O79wXuHWQJ/pVuDnO+8PNawO3SjJ3yIpaOx4EshHhcjB2L54vmHoLpmBr4Z4NprZs+8n0Pb1dXmh1elpB19j9s1vSKuwMVu13/7QwZOUjqFXePzxk6dqc2h2xf/j/AsVy4b0+OPnbf64LOd9Czs62Ab3OyBk2zBlDBDia4JyQOBXeA6ZIEBbfFkcyjmYkBHZ1+HAcXatn7LJ4+zBD8ToN5iKOcKRDKZvl66+q0LcsLzf8mxqA5+eGm1W5/s26XLrHnMgxwVwbtyJOInuCd4VRY0Rd9s4YloNm7UrLeDpcr7jSjaZQ6nPcK2GXnrNYBmldg6AJW/Ax1l7lR/tGQ47RkNlONjK1dlJhba77eK3Z9c2Td9XalRQb8Ktic2D4c8UGzk1t5hYLcN0+PE5v0AX7mupack2bM+8guRURmm4A6HKmNxc4ulzITRXWpUJrsWVx2SU2+1dVZRA6FSk5EB6rysB4w8b222FQxhitS9V0+rG9y8ZyPsdtIfGsxdMKvr5h856CBzlhN4IHVrBb8BU6gyqdgwAeQ+cTTyK37SSyz2wSeUGVx+6xiUQAZXDsqrph0A99r5OZVOu34+/sng/tfhhUXbxQdq/7+efUc5wHRvFlB6KDHQhgbJe23YAIutqAcOrZRum9xuZXJb+x0ZOqdFnLn9da3unsYQ3sai3vnNe9X8YLeuV7/09MdPPonc2tFzjqlmQVf7CZXYfeiQ0Ggo5y2SuTtdZpc9k5Cf+bjhgccMoeP979KYKPg6umZ0/vKufWYn5DcnluwnWMbX7bO1KptrEcAuZzgI63AhxwFnJ6gzT0fPj11h2h8aI4PHbdASzjVUpgqvFUa3ejx16Vdw0dM5/dw90Ovu/1Xb/iEri7VzUyGs2w3+YQGjt5jrfnIsCBFxGE0LwIH+xxcYO+W733Bf5bl2ayWPw+Lqte/MrQuf0B</diagram></mxfile>
```

Comment:
Since one bookstore can have multiple books available in it, it is not logical to make a book_id as FK in bookstore. It should be vice versa. Book table should have as and FK id of the store it is stored in
![image](https://github.com/user-attachments/assets/30e0940b-6e4d-457a-8e3d-c746be215f48)